zap/src/test/java/org/zaproxy/zap/utils/JsonUtilUnitTest.java

```diff
@@ -139,6 +139,50 @@ void shouldNotQuoteFunctionHeader() {
         assertValueAfterJsonObject(processedValue, value);
     }
 
+    @Test
+    void shouldReturnKeyStrForJsonObject() {
+        // Given
+        String json =
+                """
+                    {"aaa":"bbb", "ccc":"ddd", "eee":"fff"}
+                    """;
+
+        // When
+        String res = JsonUtil.getJsonKeyString(json);
+
+        // Then
+        assertThat(res, is(equalTo("{aaa,ccc,eee}")));
+    }
+
+    @Test
+    void shouldReturnKeyStrForJsonArray() {
+        // Given
+        String json = """
+                    ["aaa", "bbb", "ccc"]
+                    """;
+
+        // When
+        String res = JsonUtil.getJsonKeyString(json);
+
+        // Then
+        assertThat(res, is(equalTo("[]")));
+    }
+
+    @Test
+    void shouldReturnKeyStrForDeepJsonObject() {
+        // Given
+        String json =
+                """
```

Comment:
combine

Reply:
Like L160

Reply:
Trieb. It gets changed back by spotlessApply